### PR TITLE
Expose Advisory data in API

### PR DESCRIPTION
**Pull request by ykodwani01**
Solve issue #1433 
Signed-off-by: Yash Kodwani <202101418@daiict.ac.in>

### CODE REVIEW

**Comment by ykodwani01** [comment]
@ziadhany Requested Changes are done

**Comment by ziadhany** [comment]
@ykodwani01 Good Work! Please run `make valid` and `make test` before committing to run the tests and fix the code style 

and Can you try to avoid closing the pull request? you can try [force push](https://www.educative.io/answers/what-is-the-git-push---force--u-origin-command) instead

**Comment by ykodwani01** [comment]
> @ykodwani01 Good Work! Please run `make valid` and `make test` before committing to run the tests and fix the code style
> 
> and Can you try to avoid closing the pull request? you can try [force push](https://www.educative.io/answers/what-is-the-git-push---force--u-origin-command) instead

Yes will make sure from next time.

File: vulnerabilities/tests/test_api.py [hunk 1]
```diff
@@ -280,6 +303,17 @@ def test_api_with_single_vulnerability_with_filters(self):
                     "description": "The software performs operations on a memory buffer, but it can read from or write to a memory location that is outside of the intended boundary of the buffer.",
                 },
             ],
+            "advisory": [
+                {
+                    "unique_content_id": "6b7d417a552b19f26a5c2267ba7876c2",
```
Comment (ziadhany) [inline vulnerabilities/tests/test_api.py:308]: I don't think we should display the `unique_content_id`

**Comment by TG1999** [comment]
@ykodwani01 I am closing this PR for now, feel free to re-open when it's ready to merge

**Pull request closed without merging**